wallet-extension/README.md

```diff
@@ -0,0 +1,32 @@
+# <p align="center">=nil; Wallet Extension 🔐</p>
+
+Welcome to the **=nil; Wallet Extension** — a user-friendly way to manage accounts on the **=nil;** blockchain. This extension provides essential tools for interacting with the network or connecting with decentralized applications (dApps). Give it a spin and let us know what you think! 🚀
+
+---
+
+## Roadmap 🛣️
+
+- [ ] **Basic Wallet Features** ⏳
+    - Create a new wallet
+    - Send & Receive tokens
+    - Check your balance
+    - Top-up test currencies
+- [ ] **Wallet SDK for dApp Developers**
+- [ ] **Sandbox Integration**
+- [ ] **Further Improvements & Feature Requests**
+
+---
+
+## Languages 🌐
+
+By default, everything is in **English**. However, we plan to support multiple languages in the near future. We use **i18next** to handle translations, and we'd love to hear which languages you'd like to see!
+
+---
+
+## How to Use ⚙️
```

Comment:
should readme be adopted to monorepo?

Reply:
I personally feel that some of these integrations like wallet, even if they are in the same monorepo should have their "root" readme, as putting everything in the main root readme will crate too much data